packages/fuel-gauge/src/str-slice.test.ts

```diff
@@ -0,0 +1,64 @@
+import { generateTestWallet } from '@fuel-ts/account/test-utils';
+import type { WalletUnlocked } from 'fuels';
+import { Wallet, bn, Provider, FUEL_NETWORK_URL } from 'fuels';
+
+import type { PredicateStrSliceAbiInputs } from '../test/typegen';
+import {
+  PredicateStrSliceAbi__factory,
+  ScriptStrSliceAbi__factory,
+  StrSliceAbi__factory,
+} from '../test/typegen';
+import contractBytes from '../test/typegen/contracts/StrSliceAbi.hex';
+
+/**
+ * @group node
+ */
+describe('str slice', () => {
+  let provider: Provider;
+  let baseAssetId: string;
+  let sender: WalletUnlocked;
+
+  beforeAll(async () => {
+    provider = await Provider.create(FUEL_NETWORK_URL);
+    baseAssetId = provider.getBaseAssetId();
+    sender = await generateTestWallet(provider, [[500_000, baseAssetId]]);
+  });
+
+  it('echoes a str slice [CONTRACT]', async () => {
+    const contract = await StrSliceAbi__factory.deployContract(contractBytes, sender);
```

Comment:
Can you please rewrite these tests to use `launchTestNode`? IMO all new test files should use that utility so that #2579 has its scope set only to the existing files.

Reply:
Good point 👍🏻

Reply:
https://github.com/FuelLabs/fuels-ts/pull/2685/commits/6ee4d09025dde3d8117b9d6c0b45f77f5fec97ff